slimta/queue/__init__.py

```diff
@@ -313,7 +315,7 @@ def enqueue(self, envelope):
         envelopes = self._run_policies(envelope)
         ids = self._pool_imap('store', self.store.write, envelopes,
                               repeat(now))
-        results = zip(envelopes, ids)
+        results = list(six.moves.zip(envelopes, ids))
```

Comment:
I don't like "shaddowing" things for "making it look good", ("explicit is better than implicit" ;-) but if you think it fits better with slimta-style, I can for sure, let me know.